### ISSUE TITLE
Doesn't send any SMS's

### ISSUE THREAD

**Issue by CarsterMC** [TheSpeedX/TBomb#269]
The Bomber doesn't send any SMS's.
Here's some information.
I've tried running it on Termux and Windows 10.
I've also only tried it on my own phone. Maybe that's the problem? I have no idea.

**Comment by 0n1cOn3** [comment]
Which Country Code did you tried ?

**Comment by C4l1b3r** [comment]
+1 has more misses than hits and when it hits it doesn't actually send

**Comment by rhpo** [comment]
@0n1cOn3 is it also ok to NOT RECEIVE ANY MESSAGE?

(country code: 213)

**Comment by 0n1cOn3** [comment]
> @0n1cOn3 is it also ok to NOT RECEIVE ANY MESSAGE?
> 
> 
> 
> (country code: 213)
> 
> 

Yes. Thanks for providing the country code. I will add this to the supported/non-supported list on Telegram :)

**Comment by rhpo** [comment]
@0n1cOn3 can I get a telegram invite link?

**Comment by rhpo** [comment]
@0n1cOn3 Sorry I meant to say I wanted to check the supported/non-supported list

**Comment by 0n1cOn3** [comment]
> @0n1cOn3 Sorry I meant to say I wanted to check the supported/non-supported list

No Problem:

https://t.me/TBombChat/14765

**Comment by TheSpeedX** [comment]
Hello!

After careful consideration, I’ve decided to close all open issues for this repository. 
This project has grown into something beyond my initial expectations, but I’ve found it increasingly difficult to maintain due to low-quality contributions and spam pull requests/issues. Furthermore, I am transitioning to new ventures, and I won’t be able to continue contributing to TBomb any further.

Though I’m stepping away, I’m grateful to all those who have used and supported this tool. TBomb will remain usable through Git and PyPI, but I won’t be actively involved moving forward.

Thank you for your understanding and for using this tool!
Best regards,
TheSpeedX
